src/app/clusters/push-av-stream-transport-server/push-av-stream-transport-server.cpp

```diff
@@ -0,0 +1,506 @@
+            handlerContext,
+            [this](HandlerContext & ctx, const auto & commandData) { HandleAllocatePushTransport(ctx, commandData); });
+
+        break;
+
+    case Commands::DeallocatePushTransport::Id:
+        ChipLogDetail(Zcl, "PushAVStreamTransport: Deallocating Push Transport");
+
+        HandleCommand<Commands::DeallocatePushTransport::DecodableType>(
+            handlerContext,
+            [this](HandlerContext & ctx, const auto & commandData) { HandleDeallocatePushTransport(ctx, commandData); });
+
+        break;
+
+    case Commands::ModifyPushTransport::Id:
+        ChipLogDetail(Zcl, "PushAVStreamTransport: Modifying Push Transport");
+
+        HandleCommand<Commands::ModifyPushTransport::DecodableType>(
+            handlerContext,
+            [this](HandlerContext & ctx, const auto & commandData) { HandleModifyPushTransport(ctx, commandData); });
+
+        break;
+
+    case Commands::SetTransportStatus::Id:
+        ChipLogDetail(Zcl, "PushAVStreamTransport: Setting Push Transport Status");
+
+        HandleCommand<Commands::SetTransportStatus::DecodableType>(
+            handlerContext, [this](HandlerContext & ctx, const auto & commandData) { HandleSetTransportStatus(ctx, commandData); });
+
+        break;
+
+    case Commands::ManuallyTriggerTransport::Id:
+        ChipLogDetail(Zcl, "PushAVStreamTransport: Manually Triggered Push Transport");
+
+        HandleCommand<Commands::ManuallyTriggerTransport::DecodableType>(
+            handlerContext,
+            [this](HandlerContext & ctx, const auto & commandData) { HandleManuallyTriggerTransport(ctx, commandData); });
+
+        break;
+
+    case Commands::FindTransport::Id:
+        ChipLogDetail(Zcl, "PushAVStreamTransport: Finding Push Transport");
+
+        HandleCommand<Commands::FindTransport::DecodableType>(
+            handlerContext, [this](HandlerContext & ctx, const auto & commandData) { HandleFindTransport(ctx, commandData); });
+
+        break;
+    default:
+        // Mark unrecognized command as UnsupportedCommand
+        handlerContext.mCommandHandler.AddStatus(handlerContext.mRequestPath, Status::UnsupportedCommand);
+        break;
+    }
+}
+
+TransportConfigurationStruct * PushAvStreamTransportServer::FindStreamTransportConnection(const uint16_t connectionID)
+{
+    for (auto & transportConnection : mCurrentConnections)
+    {
+        if (transportConnection.connectionID == connectionID)
+            return &transportConnection;
+    }
+    return nullptr;
+}
+
+uint16_t PushAvStreamTransportServer::GenerateConnectionID()
+{
+    static uint16_t lastAssignedConnectionID = 0;
+    uint16_t nextConnectionID;
+
+    do
+    {
+        if (lastAssignedConnectionID == MAX_PUSH_TRANSPORT_CONNECTION_ID)
+            nextConnectionID = 0;
+        else
+            nextConnectionID = static_cast<uint16_t>(lastAssignedConnectionID + 1);
+    } while (FindStreamTransportConnection(nextConnectionID));
```

Comment:
Use actual booleans for a boolean expression.
Suggest: while (FindStreamTransportConnection(nextConnectionID) != nullptr);